lib/cn/cndb_internal.h

```diff
@@ -278,10 +280,11 @@ struct cndb_txm {
 
 struct cndb_txd {
     struct cndb_hdr hdr;
-    u64             mtd_cnid;
-    u64             mtd_id;
-    u64             mtd_tag;
-    u32             mtd_n_oids;
+    uint64_t        mtd_cnid;
+    uint64_t        mtd_id;
+    uint64_t        mtd_tag;
+    uint64_t        mtd_hoid;
+    uint32_t        mtd_n_oids;
```

Comment:
Can you put the _hoid field in the same place in both structs, i.e., right after the _tag fields?